src/components/DayInformation/DayInformation.tsx

```diff
@@ -16,6 +16,9 @@ function DayInformation({ day }: Props) {
     const [dayTasks, setDayTasks] = useState<Task[]>([]);
     const [recurringDayTasks, setRecurringDayTasks] = useState<RecurringTask[]>([]);
     const [newTaskOpen, setNewTaskOpen] = useState<boolean>(false);
+    const [isEditing, setIsEditing] = useState<string>('');
```

Comment:
the name isEditing would represent a boolean
instead you can call it editingTaskId